pkg/util/utils_test.go

```diff
@@ -259,6 +259,15 @@ func TestValidateSysctlBadSysctl(t *testing.T) {
 	assert.Error(t, err)
 }
 
+func TestValidateSysctlBadSysctlWithExtraSpaces(t *testing.T) {
+	strSlice := []string{
+		"net.ipv4.ping_group_range = 0 0",
+		"net.ipv4.ping_group_range=0 0 ",
+	}
+	_, err := ValidateSysctls(strSlice)
+	assert.Error(t, err)
+}
```

Comment:
Might be good to add a test that should pass, perhaps:

```
	strSlice := []string{
		" net.ipv4.ping_group_range=0 0          ",
		"          net.ipv4.ping_group_range=0 0 ",
	}
```
Also, can you add a check for the error text you're expecting, or at least "extra spaces found" as the substring?

Reply:
@TomSweeneyRedHat thanks for suggestions. I was thinking that other tests for `ValidateSysctls` would still go through my changes, as long as we don't see regressions on the other existing tests I think there's no need to introduce other test cases? I can still add a few if you wish.

Yes, I'll update my PR to check the message as well.

Reply:
Also, this won't be (and shouldn't be, IMO) passing, as we do `arr := strings.Split(val, "=")` in `ValidateSysctls`.

```
    strSliceGood := []string{                                                    
        " net.ipv4.ping_group_range=0 0          ",                              
        "          net.ipv4.ping_group_range=0 0 ",                             
    }                                                                            
    _, err := ValidateSysctls(strSliceGood)                                     
    assert.Nil(t, err)
```